### ISSUE TITLE
[Question] source for geotag coordinates?

### ISSUE THREAD

**Issue by HannimalCrackers** [open-editions/corpus-joyce-portrait-TEI#89]
Hi @goldieshen 

I'm doing some work mapping locations and I have a couple of questions.

(1) there are some locations I'd like to add/edit. So that I can be consistent, what is the source of the latitude and longitude coordinates you used?

(2) sometimes <placeName> appears with an accompanying <place> tag, sometimes <placeName> appears without a <place> tag. Is there a structural logic I should be following that drives when the <place> tag does and doesn't appear?

Thank you!
Hannah

**Comment by goldieshen** [comment]
Dear Hannah,

For the longitude and latitudes, I went through the book and input the
names on online sites like this:
https://www.latlong.net/
I don’t remember the exact sites I used, but this is probably the one I
used.

For tags, I don’t remember what I used them for, but I tagged locations in
the book to which the protagonist both physically travelled and merely
alluded (faraway continents, etc.). Perhaps the tags clarify whether the
location is actually part of the trajectory of the book or whether it is
just part of the book’s consciousness/scope.

Good luck on the project, Hannah! If you feel like tags are confusing, you
can reorganize them. Feel free to edit anything, and thank you!

Best wishes,
Goldie
